packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.selectors.ts

```diff
@@ -39,6 +39,11 @@ export const selectorChartRawYAxis = createSelector(
  * Following selectors are not exported because they exist in the MIT chart only to ba able to reuse the Zoom state from the pro.
  */
 
+export const selectorChartZoomIsInteracting = createSelector(
+  selectorChartZoomState,
+  (zoom) => zoom?.isInteracting,
+);
+
```

Comment:
The selector got moved to the communit packeg to be able to stop voronoi computation during the zoom